### DISCUSSION TITLE
DMV DISCOVER_CALC_DEPENDENCY breaks at seemingly random moments

### DISCUSSION THREAD

**Discussion by Remko1981** [TabularEditor/TabularEditor3#996] [🙋♀️TE3 Q&A]
We are automating a lot of work for our SSAS instance. For that purpose we extract some DMV's. But for some reason the DMV DISCOVER_CALC_DEPENDENCY breaks at seemingly random moments. The DMV cannot be queried anymore and when i try to do this with DAX studio it even locks up the SSAS server requiring a reboot. Deleting the SSAS database and redeploying seems to be the only way to fix the problem. Google has a few other reports about this happening but no solution.

Does anyone have a clue what might be happening? Or does anyone know of a fix which does not require dropping and redeploying? Or maybe even how to use an other DMV to get around this one?

**Comment by otykier** [comment]
You should raise a support ticket with Microsoft, to have them investigate what’s going on.
